docs/references/production_metrics.md

```diff
@@ -142,4 +146,28 @@ Then you can access the Grafana dashboard at http://localhost:3000.
 
 ### Grafana Dashboard
 
+In a new Grafana setup, ensure that you have the `Prometheus` data source enabled. To check that, go to `http://localhost:3000/connections/datasources` and ensure that `Prometheus` is enabled.
```

Comment:
is there a way to prepare a config file (or list of commands in Dockerfile) that gets the datasource set up, then supply it to the grafana container, and get the entire thing ready to use end to end?

I feel that requiring a manual step would be error prone and difficult for the team to troubleshoot for users. we're targeting users with no metrics knowledge (or no time to to spend on metrics set up) so "ready-to-use"ness is important.

Reply:
let me see what I can do with a `grafana.ini`